Renegade X Launcher/Views/ApplyUpdateWindow.xaml.cs

```diff
@@ -68,17 +70,26 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
                 return "pending";
             else if (progress.State == DirectoryPatchPhaseProgress.States.Indeterminate)
             {
-                //var unitAndScale = UnitAndScale.GetPreferredByteFormat(progress.Size.Total);
-                //return string.Format("{0} / ~{1} {2}", unitAndScale.GetFormatted(progress.Size.Done), unitAndScale.GetFormatted(progress.Size.Total), unitAndScale.Unit);
                 double perc = ((double)progress.Size.Done / (double)progress.Size.Total) * 100.00; ;
-                return string.Format("{0}%", perc.ToString("0.##"));
+                return $"{perc:0.##}%";
+            }
+            else if ( progress.DownloadThreads == 0 )

```

Comment:
We should really stick to using the state code, rather than checking the download threads manually here